src/GovUk.Education.ExploreEducationStatistics.Content.Model/Services/Interfaces/IMethodologyService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.ViewModels;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model.Services.Interfaces
+{
+    public interface IMethodologyService
+    {
+        List<ThemeTree> GetTree();
+
+        Methodology Get(string slug);
```

Comment:
Get by slug isn't used anymore (only by the service test).

Reply:
switched to Id as this may be useful down the line